### PR TITLE
Sharded travis setup

### PR DESCRIPTION
**Pull request by snopoke**
test setup for sharding

requires moving back to non-containerized builds :cry:

### CODE REVIEW

**Comment by czue** [comment]
> requires moving back to non-containerized builds

one upshot if we do this is that we can easily add elasticsearch tests

**Comment by snopoke** [comment]
Resources:
* https://github.com/petere/plxslt/blob/master/.travis.yml
* https://gist.githubusercontent.com/petere/6023944/raw/pg-travis-test.sh
* https://github.com/pgRouting/pgrouting/blob/master/tools/travis/pgrouting_prepare.sh

File: .travis.yml [hunk 1]
```diff
@@ -12,10 +12,12 @@ env:
...
     - COUCHDB_PW=""
     - TRAVIS_INSTALL="y"
+    - PGVERSION="9.1"
   matrix:
-    - MATRIX_TYPE=javascript BOWER=yes
-    - MATRIX_TYPE=python TESTRUNNER=testrunner.GroupTestRunnerCatchall BOWER=yes
-    - MATRIX_TYPE=python TESTRUNNER=testrunner.GroupTestRunner0
+#    - MATRIX_TYPE=javascript BOWER=yes
+#    - MATRIX_TYPE=python TESTRUNNER=testrunner.GroupTestRunnerCatchall BOWER=yes
+#    - MATRIX_TYPE=python TESTRUNNER=testrunner.GroupTestRunner0
```
Comment (benrudolph) [inline .travis.yml]: think we should uncomment these

**Comment by benrudolph** [comment]
this is great and serves as a great example for how to setup our own sharded db. yay sharded :elephant: :elephant: :elephant:

File: .travis/kafka_setup.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -ev
+
+# kafka install, copied from https://github.com/wvanbergen/kafka/blob/master/.travis.yml
```
Comment (benrudolph) [inline .travis/kafka_setup.sh]: i think if you merge in master, this isn't necessary anymore

**Comment by snopoke** [comment]
Closed in favour of https://github.com/dimagi/commcare-hq/pull/10097

**Pull request closed without merging**